src/util/chromium-preferences.js

```diff
@@ -0,0 +1,33 @@
+import { UsageError } from '../errors.js';
+
+export function coerceCLICustomChromiumPreference(cliPrefs) {
+  const customPrefs = {};
```

Comment:
```suggestion
  const customPrefs = new Map();
```

Suggesting a few changes to avoid prototype pollution.